generate/templates/partials/convert_to_v8.cc

```diff
@@ -61,7 +61,7 @@
     {% if cppClassName == 'Wrapper' %}
       to = {{ cppClassName }}::New({{= parsedName =}});
     {% else %}
-      to = {{ cppClassName }}::New({{= parsedName =}}, {{ selfFreeing|toBool }});
+      to = {{ cppClassName }}::New({{= parsedName =}}, {{ selfFreeing|toBool }} {% if shouldDuplicate %}, true{% endif %});
```

Comment:
Tried it, but there is a case where we call that on something that I think is a struct.  And I didn't want to add that parameter to struct constructors without adding support for it as well.